packages/vscode-extension/src/webview/utilities/consts.ts

```diff
@@ -75,3 +75,19 @@ export const AndroidSupportedDevices: DeviceProperties[] = [
     maskImage: pixel6amask,
   },
 ] as const;
+
+
+function getPlatform() {
+  // https://stackoverflow.com/a/73619128
+  if (typeof navigator.userAgentData !== 'undefined' && navigator.userAgentData != null) {
```

Comment:
This function has a lot of redundant logic.

```ts
function getPlatform() {
  return navigator.userAgentData?.platform ?? navigator.platform ?? 'unknown';
}
```

should have the same semantics.

Reply:
In new `Platform` class your simpler version is used.